guaranteed-message-delivery/message_consumer.bal

```diff
@@ -0,0 +1,82 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/rabbitmq;
+import ballerinax/twilio;
+
+type Doctor record {|
+    string name;
+    string hospital;
+    string category;
+    string availability;
+    decimal fee;
+|};
+
+type ReservationResponse record {|
+    int appointmentNumber;
+    Doctor doctor;
+    Patient patient;
+    string hospital;
+    boolean confirmed;
+    string appointmentDate;
+|};
+
+type MessageContent record {|
+    *ReservationRequest;
+    string category;
+|};
+
+type RabbitMqMessage record {|
+    *rabbitmq:AnydataMessage;
+    MessageContent content;
+|};
+
+configurable string smsFromNumber = ?;
+configurable string twillioAccSId = ?;
+configurable string twillioAuthToken = ?;
+
+final http:Client hospitalBackend = check new ("http://localhost:9090/");
+
+final twilio:Client twilioEp = check new ({
+    twilioAuth: {
+        accountSId: twillioAccSId,
+        authToken: twillioAuthToken
+    }
+});
+
+@rabbitmq:ServiceConfig {
+    queueName: "ReservationQueue",
+    autoAck: false
```

Comment:
Can't we use autoAck true and simplify the code?

Reply:
Wanted to show a usage manual ack. Anyway changed to auto-ack true